Mathlib/RingTheory/UniqueFactorizationDomain.lean

```diff
@@ -878,17 +875,14 @@ protected noncomputable def normalizationMonoid : NormalizationMonoid α :=
       apply normalizedFactors_prod hx)
 #align unique_factorization_monoid.normalization_monoid UniqueFactorizationMonoid.normalizationMonoid
 
-noncomputable instance : Inhabited (NormalizationMonoid α) :=
-  ⟨UniqueFactorizationMonoid.normalizationMonoid⟩
-
 end UniqueFactorizationMonoid
 
 namespace UniqueFactorizationMonoid
 
 variable {R : Type*} [CancelCommMonoidWithZero R] [UniqueFactorizationMonoid R]
 
 theorem no_factors_of_no_prime_factors {a b : R} (ha : a ≠ 0)
```

Comment:
How about upgrading this to an iff (and fixing the name):
```suggestion
theorem isRelPrime_iff_forall_dvd_not_prime_of_ne_zero {a b : R} (ha : a ≠ 0) :
    IsRelPrime a b ↔ ∀ ⦃d⦄, d ∣ a → d ∣ b → ¬ Prime d :=
```

Reply:
There are plenty of lemmas with `no_...factors` in the name and I think they're pretty intuitive and almost established a certain kind of convention, so I'm only changed `no_factors` to `isRelPrime` and upgrade to iff. Notice that [isCoprime_of_prime_dvd](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/PrincipalIdealDomain.html#isCoprime_of_prime_dvd) etc. are similar lemmas that are not iff.

Reply:
Given that this would be a rename, I certainly don't insist but I do think these `no_...factors` names should be tidied up one day.

Reply:
Oh I see you implemented this suggestion 🎉